### DISCUSSION TITLE
Customize WindowCommands icons

### DISCUSSION THREAD

**Discussion by borioan** [fluentribbon/Fluent.Ribbon#1182] [Q&A] [answered]
Hello!
I like to change the size and the icons of close / minimize and maximize buttons.
It is possible? 
Appreciate any insights on this. Thanks!

**Comment by batzen** [comment, marked as answer]
Sorry for my late response.

Currently there is no "easy" way for doing so.
To achieve what you want you currently have to copy the whole template `Fluent.Ribbon.Templates.WindowCommands` from https://github.com/fluentribbon/Fluent.Ribbon/blob/develop/Fluent.Ribbon/Themes/Controls/WindowCommands.xaml and change the content of `PART_Min` etc. to your needs.

**Reply by borioan** [reply]
Thank you very much!!
